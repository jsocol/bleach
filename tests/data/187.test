<div id="120"><script>
function makePopups(){
    for (i=1;i<6;i++) {
        window.open('popup.html','spam'+i,'width=50,height=50');
    }
}
</script>
<body>
<a href="#" onclick="makePopups()">Spam</a>//["'`-->]]>]</div>
