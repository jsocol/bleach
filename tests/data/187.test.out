<div id="120">

<a href="#">Spam</a>//["'`--&gt;]]&gt;]</div>
